src/inc/regdisp.h

```diff
@@ -430,6 +423,10 @@ inline void FillRegDisplay(const PREGDISPLAY pRD, PT_CONTEXT pctx, PT_CONTEXT pC
     pRD->ctxPtrsOne.Esi = &pctx->Esi;
     pRD->ctxPtrsOne.Edi = &pctx->Edi;
     pRD->ctxPtrsOne.Ebp = &pctx->Ebp;
+
```

Comment:
A nit - for consistency with other target, you could now revert back to using a loop for copying the registers if you reordered them in the KNONVOLATILE_CONTEXT_POINTERS to match their order in the CONTEXT.